wasmbinding/test/custom_query_test.go

```diff
@@ -79,13 +79,15 @@ func TestFinalizedEpoch(t *testing.T) {
 	require.Equal(t, resp.EpochInfo.EpochNumber, uint64(0))
 	require.Equal(t, resp.EpochInfo.LastBlockHeight, uint64(0))
 
-	epoch := babylonApp.EpochingKeeper.InitEpoch(ctx)
-	babylonApp.CheckpointingKeeper.SetCheckpointFinalized(ctx, epoch.EpochNumber)
+	err := babylonApp.EpochingKeeper.InitEpoch(ctx, nil)
+	require.NoError(t, err)
+	expEpochNum := uint64(0)
```

Comment:
What is this testing ? :thinking:

Reply:
seems to be testing a contract query related to the epoch